### DISCUSSION TITLE
Conbee 3 released. Is it worth changing Conbee 2 to Conbee 3?

### DISCUSSION THREAD

**Discussion by KitHubek** [Koenkk/zigbee2mqtt#19883] [General]
Hello. 

Iam using HA with zigbee2mqtt integration over 2 years (Everything working fine) 
My coordinator is Conbee 2 and have 133 zigbee devices connected (67 routers and rest battery powered)
As we know, last time, Conbee 3 released
I'm thinking about changing conbee 2 to conbee 3
but, does it make sense? is it worth it?

1. Will just switching to conbee 3 require connecting all devices to the network again? With the number I have - 133, it would be problematic.
2. Does zigbee2mqtt fully support conbee 3? Are any special configurations required for this coordinator?
3. What benefits will there be after switching to conbee 3, if any?
4. Is there any point in changing conbee 2 to conbee 3 if currently conbee 2 works without any problems?

thanks

![conbee3-header-en](https://github.com/Koenkk/zigbee2mqtt/assets/91805313/0146e5a3-c2ac-492c-b4f7-d366b6e1841c)

**Comment by riesi77** [comment]
ConBee III is based on a different chipset (EFR32MG21), so it is similar to the SONOFF ZBDongle-E. You will have to re-pair all devices if you change the coordinator to one with a different chipset. 
Here is the status on the current support of EFR32MG21 based chips: https://github.com/Koenkk/zigbee2mqtt/discussions/19812

**Comment by KitHubek** [comment]
thanks. read this.

but main qustions are:

What benefits will there be after switching from conbee 2 to conbee 3, if any?
Is there any point in changing conbee 2 to conbee 3 if currently conbee 2 works without any problems?

**Reply by Silvest89** [reply]
Why would you change if everything works fine as it is now? Since you don't know about the benefits, don't bother changing. You want to change it just for the sake of because you can and not because you need to.

**Reply by KitHubek** [reply]
ehh. v.g. ansver

**Reply by hioctane61** [reply]
I once thought Windows XP worked well and why move onto Windows 7, if it's "grand"? Hindsight is 20/20 so can see that was a naïve point of view. There's no standing still with technology, you either move forward or back..

**Comment by daniu-de** [comment]
Question still not answered ...

What are the benefits?

I think the comparison with windows is funny but: who would not be able to list some benefits of win 7 compared to win xp?  If there were none it would never have made sense to switch in that case either. Changing because of something is new does only make sense to the vendor 😁

**Reply by smrose** [reply]
It you look at the "what's new" section of the product page, there is some information that might be compelling to some. Specifically, there is this:  "OpenThread Border Router firmware is offered as an alternative application." But I wouldn't expect it to be compelling to you, as you have twelve dozen Zigbee devices.

**Comment by SanderKo85** [comment]
hi! Searching for the same answers here :)

**Comment by chris-1243** [comment]
If your system is running and stable why would you change something ?

If you plan to run multi-rcp firmware with it might be a choice.

Otherwise, it may look promising on the paper but with some potentials downside like instability.

**Comment by daniu-de** [comment]
It is not just interesting for people who own conbee ii and consider switching to conbee iii but also people who think about buying a conbee first time ... conbee III is more expensive and conbee ii easy to buy used

**Comment by Marx2** [comment]
maybe it doesn't need to live on cable extender, and you can put it directly to usb port?
